<html>
<head>
<title>TestNG:   Spentra </title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'> Spentra </h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>8/4/0</td>
</tr><tr>
<td>Started on:</td><td>Mon Jan 25 09:03:13 IST 2021</td>
</tr>
<tr><td>Total time:</td><td>484 seconds (484658 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.functional.tests.AddEmp.addEmployeeDetails()'><b>addEmployeeDetails</b><br>Test class: com.functional.tests.AddEmp<br>Parameters: 2, y, test_Auto, Jon, Smith, 2222, 9876543212, testauto1@mailinator.com, 2527 Decatur, Apt 40, Denver, 12345</td>
<td><div><pre>java.lang.AssertionError: expected [Employee profile with this Employee ID already exists.] but found [Employee added successfully.]
	at com.pages.AddEmpPage.verifyMessage(AddEmpPage.java:170)
	at com.functional.tests.AddEmp.addEmployeeDetails(AddEmp.java:34)
	at com.framework.Setup.executeTest(Setup.java:39)
	at com.framework.Start.main(Start.java:23)
... Removed 26 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace275448142", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace275448142'><pre>java.lang.AssertionError: expected [Employee profile with this Employee ID already exists.] but found [Employee added successfully.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at com.pages.AddEmpPage.verifyMessage(AddEmpPage.java:170)
	at com.functional.tests.AddEmp.addEmployeeDetails(AddEmp.java:34)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at com.framework.Setup.executeTest(Setup.java:39)
	at com.framework.Start.main(Start.java:23)
</pre></div></td>
<td>55</td>
<td>com.functional.tests.AddEmp@56e07a08</td></tr>
<tr>
<td title='com.functional.tests.AddEmp.addEmployeeDetails()'><b>addEmployeeDetails</b><br>Test class: com.functional.tests.AddEmp<br>Parameters: 3, y, test_Automation, Josh, Miller, 3333, 8273979918, testauto7@mailinator.com, 6567 Decatur, Apt90, Denver, 67890</td>
<td><div><pre>java.lang.AssertionError: expected [Employee profile with this Employee ID already exists.] but found [Employee added successfully.]
	at com.pages.AddEmpPage.verifyMessage(AddEmpPage.java:170)
	at com.functional.tests.AddEmp.addEmployeeDetails(AddEmp.java:34)
	at com.framework.Setup.executeTest(Setup.java:39)
	at com.framework.Start.main(Start.java:23)
... Removed 26 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace7064297", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace7064297'><pre>java.lang.AssertionError: expected [Employee profile with this Employee ID already exists.] but found [Employee added successfully.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at com.pages.AddEmpPage.verifyMessage(AddEmpPage.java:170)
	at com.functional.tests.AddEmp.addEmployeeDetails(AddEmp.java:34)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at com.framework.Setup.executeTest(Setup.java:39)
	at com.framework.Start.main(Start.java:23)
</pre></div></td>
<td>30</td>
<td>com.functional.tests.AddEmp@56e07a08</td></tr>
<tr>
<td title='com.functional.tests.AddEmp.addEmployeeDetails()'><b>addEmployeeDetails</b><br>Test class: com.functional.tests.AddEmp<br>Parameters: 4, y, test_Automation1, Jessy, David, 4444, 9809765555, testauto5@mailinator.com, 1002 Decatur, Apt67, Denver, 98765</td>
<td><div><pre>java.lang.AssertionError: expected [Employee profile with this Employee ID already exists.] but found [Employee added successfully.]
	at com.pages.AddEmpPage.verifyMessage(AddEmpPage.java:170)
	at com.functional.tests.AddEmp.addEmployeeDetails(AddEmp.java:34)
	at com.framework.Setup.executeTest(Setup.java:39)
	at com.framework.Start.main(Start.java:23)
... Removed 26 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1427396510", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1427396510'><pre>java.lang.AssertionError: expected [Employee profile with this Employee ID already exists.] but found [Employee added successfully.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at com.pages.AddEmpPage.verifyMessage(AddEmpPage.java:170)
	at com.functional.tests.AddEmp.addEmployeeDetails(AddEmp.java:34)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at com.framework.Setup.executeTest(Setup.java:39)
	at com.framework.Start.main(Start.java:23)
</pre></div></td>
<td>63</td>
<td>com.functional.tests.AddEmp@56e07a08</td></tr>
<tr>
<td title='com.functional.tests.AddEmp.addEmployeeDetails()'><b>addEmployeeDetails</b><br>Test class: com.functional.tests.AddEmp<br>Parameters: 5, y, test_Automation2, Ava, Jonson, 5555, 7654984567, testauto4@mailinator.com, 8888 Decatur, Apt03, Denver, 54321</td>
<td><div><pre>java.lang.AssertionError: expected [Employee profile with this Employee ID already exists.] but found [Employee added successfully.]
	at com.pages.AddEmpPage.verifyMessage(AddEmpPage.java:170)
	at com.functional.tests.AddEmp.addEmployeeDetails(AddEmp.java:34)
	at com.framework.Setup.executeTest(Setup.java:39)
	at com.framework.Start.main(Start.java:23)
... Removed 26 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1101149296", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1101149296'><pre>java.lang.AssertionError: expected [Employee profile with this Employee ID already exists.] but found [Employee added successfully.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at com.pages.AddEmpPage.verifyMessage(AddEmpPage.java:170)
	at com.functional.tests.AddEmp.addEmployeeDetails(AddEmp.java:34)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at com.framework.Setup.executeTest(Setup.java:39)
	at com.framework.Start.main(Start.java:23)
</pre></div></td>
<td>53</td>
<td>com.functional.tests.AddEmp@56e07a08</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='com.functional.tests.Login.logintoApplication()'><b>logintoApplication</b><br>Test class: com.functional.tests.Login</td>
<td></td>
<td>18</td>
<td>com.functional.tests.Login@1b0a7baf</td></tr>
<tr>
<td title='com.functional.tests.AddEmp.logoutFromApplication()'><b>logoutFromApplication</b><br>Test class: com.functional.tests.AddEmp</td>
<td></td>
<td>16</td>
<td>com.functional.tests.AddEmp@56e07a08</td></tr>
<tr>
<td title='com.functional.tests.AddEmp.navigateToAddEmployee()'><b>navigateToAddEmployee</b><br>Test class: com.functional.tests.AddEmp</td>
<td></td>
<td>19</td>
<td>com.functional.tests.AddEmp@56e07a08</td></tr>
<tr>
<td title='com.functional.tests.Login.negativeLoginTest()'><b>negativeLoginTest</b><br>Test class: com.functional.tests.Login<br>Parameters: 2, y, cgd, 435</td>
<td></td>
<td>28</td>
<td>com.functional.tests.Login@1b0a7baf</td></tr>
<tr>
<td title='com.functional.tests.Login.negativeLoginTest()'><b>negativeLoginTest</b><br>Test class: com.functional.tests.Login<br>Parameters: 3, y, test@wrr.com, </td>
<td></td>
<td>11</td>
<td>com.functional.tests.Login@1b0a7baf</td></tr>
<tr>
<td title='com.functional.tests.Login.negativeLoginTest()'><b>negativeLoginTest</b><br>Test class: com.functional.tests.Login<br>Parameters: 4, y, , Testrig_123</td>
<td></td>
<td>25</td>
<td>com.functional.tests.Login@1b0a7baf</td></tr>
<tr>
<td title='com.functional.tests.Login.negativeLoginTest()'><b>negativeLoginTest</b><br>Test class: com.functional.tests.Login<br>Parameters: 5, y, test101@mailinator.com, </td>
<td></td>
<td>16</td>
<td>com.functional.tests.Login@1b0a7baf</td></tr>
<tr>
<td title='com.functional.tests.AddEmp.verifyEmployeeDetails()'><b>verifyEmployeeDetails</b><br>Test class: com.functional.tests.AddEmp</td>
<td></td>
<td>142</td>
<td>com.functional.tests.AddEmp@56e07a08</td></tr>
</table><p>
</body>
</html>